### ISSUE TITLE
 Error "Can't Connect . . . " After Scanning ISBN Bar Code

### ISSUE THREAD

**Issue by jasegal** [eleybourn/Book-Catalogue#808]
I hope this is in the right place; please move it if not.

After a long hiatus for unrelated reasons, I'm back to Book Catalogue again. I downloaded the latest version (I don't remember which version I had been using before my phone needed a factory reset) and imported my book database from SD card. I really like the program, but I'm experiencing a problem when I scan ISBN bar codes. The scan is successful, but immediately afterward, I get the message:

"_Sorry Can't connect to get the latest prices and offers right now. we're sorry for the inconvenience_"

Clicking the OK button in the error window takes me back to the main book list, not the book entry. The just-scanned book was correctly entered.

I assume this issue is caused by some problem in connecting to Amazon or whoever.  I searched for settings related to this, but can't seem to find any (are there any?). I also read through the Wiki and the rest of this project to no avail. I don't recall ever seeing this error before.

**More Details:**
- Samsung Galaxy SII Skyrocket i727 (AT&T version) 
- Consumer Cellular (AT&T network, mostly)
- Android 4.1.2
- BookCatalogue 5.2.2
- All of this was done while on WiFi and the app doesn't otherwise complain about connectivity.
- I successfully connected to GoodReads and LibraryThing.

I hope someone can help me get past this.

Thanks,

Jack

**Comment by Grunthos** [comment]
Thanks for the report.

Please repeat the steps to produce the error, then go to "Help"->"Send Information"

**Comment by jasegal** [comment]
OK, I seem to have resolved my own issue. While wandering around in the various settings yet again, I realized that there are choices for the zxing scanner and for a zxing COMPATIBLE scanner. Since I had compatible selected, I installed the ACTUAL zxing scanner apk and changed that setting. Suddenly, everything works just fine. I'm not sure which scanner I was actually using, but it's probably not as compatible with zxing as it needs to be.

Jack
